db/migrate/20170812152912_add_unique_constraint_to_craftsmen_employment_id.rb

```diff
@@ -0,0 +1,18 @@
+class AddUniqueConstraintToCraftsmenEmploymentId < ActiveRecord::Migration
+  def change
```

Comment:
I did that initially, but the migrations were not actually running the SQL, and no effects where present in the database. This is also the way it is done in the rails documentation for 4.0.*.

It might be deprecated in resent rails versions.